english/eat-tacos/eat-delicious-tacos

```diff
@@ -0,0 +1,7 @@
+You found a place in the street, you were still hungry and you didn't have a lot of money. Then you decided to eat in that place.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.